libycp/src/Import.cc

```diff
@@ -108,6 +108,15 @@ Import::import (const string &name, Y2Namespace *preloaded_namespace)
 	    }
 	}
 
+        // for runtime error during loading lets act like if module is there, just error namespace is created
+        if (isErrorNamespace(name_space))
+        {
+            ycp2error("Runtime error when loading module '%s'", cname);
+            m_active_modules.insert (std::make_pair (m_name, name_space));			// insert to list of known modules
+	    m_module = m_active_modules.find (m_name);
+            return 0;
```

Comment:
Why not -1? In the constructor, it says 
```cxx
     if (import (name, preloaded_namespace) != 0)
 	m_name = SymbolEntry::emptyUstring;			// mark as failed import
```

Please add a comment to the header file, explaining what the return values 0 and -1 mean.